runtime/tests/main.rs

```diff
@@ -15,44 +15,30 @@ fn register_project() {
         .create_checkpoint(&alice, project_hash, None)
         .wait()
         .unwrap();
-    let project_id = (
-        ProjectName::from_string("NAME".to_string()).unwrap(),
-        ProjectDomain::from_string("DOMAIN".to_string()).unwrap(),
-    );
-    let tx_applied = client
-        .submit(
-            &alice,
-            RegisterProjectParams {
-                id: project_id.clone(),
-                description: "DESCRIPTION".to_string(),
-                img_url: "IMG_URL".to_string(),
-                checkpoint_id,
-            },
-        )
-        .wait()
-        .unwrap();
+    let params = random_register_project_params(checkpoint_id);
+    let tx_applied = client.submit(&alice, params.clone()).wait().unwrap();
 
     let project = client
-        .get_project(project_id.clone())
+        .get_project(params.clone().id)
```

Comment:
This is more efficient:
```suggestion
        .get_project(params.id.clone())
```